fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -5074,6 +5113,31 @@ public void replayRenameColumn(TableInfo tableInfo) throws DdlException {
         throw new DdlException("not implmented");
     }
 
+    public void modifyTableDynamicPartition(Database db, OlapTable table, Map<String, String> properties) throws DdlException {
```

Comment:
The logic of modify dynamic partition is too simple.
There should be more check to validate the request.
For example, do we allow user to change the prefix of dynamic partition's name? If yes, how to make sure that your partition checking logic in `DynamicPartitionScheduler` works well after change?
As I know, You are judging whether to add the partition based on whether the partition‘ name exists.

Reply:
I first want to judge whether there exist partition which range is same as add partition, but I am not sure how to judge the two partition range, I will check it later.

Reply:
> The logic of modify dynamic partition is too simple.
> There should be more check to validate the request.
> For example, do we allow user to change the prefix of dynamic partition's name? If yes, how to make sure that your partition checking logic in `DynamicPartitionScheduler` works well after change?
> As I know, You are judging whether to add the partition based on whether the partition‘ name exists.

I use another way to judge whether the partition exists, but I am not sure if it is useful, because I am not family with code about partition.